### DISCUSSION TITLE
Problem with the dollar symbol $

### DISCUSSION THREAD

**Discussion by TrabelsiHoussem** [prisma/prisma#13056] [Q&A]
### Bug description

```
 const findUser: User = await this.users.findFirst({
      where: { firstName: '$foo' },
    });
```
when i run  the above request i got a problem which is findUser is null , if i change  on the database the firstName too "foo" 
i get the wanted result .

### Prisma information

const findUser: User = await this.users.findFirst({
      where: { firstName: '$foo' },
    });

### Environment & setup

- OS: 
- Database: 
- Node.js version: 
Ubuntu
Mongodb
node v16.13.1

### Prisma Version

```
3.12.0
```

**Comment by janpio** [comment]
What is the actual value of the `Users.firstName` in the database, `foo` or `$foo`?

**Comment by TrabelsiHoussem** [comment]
i tried both  , the first one works fine but the second one doesn't work

**Comment by janpio** [comment]
If the database has an entry where `firstName` is `foo` that is very much expected - a `where` search returns back what you are searching for. That is the point of it.

**Reply by TrabelsiHoussem** [reply]
i m searching  for an exist entry firstName is $foo  but it returns null

**Reply by janpio** [reply]
What database and version of that database are you using? Then we can try to reproduce this.

**Reply by TrabelsiHoussem** [reply]
i m using mongodb on the  cloud .

**Reply by janpio** [reply]
Ok. And what version of MongoDB is running on that cloud?

**Reply by TrabelsiHoussem** [reply]
its 5.0.8

**Reply by janpio** [reply]
Ok, so the bug report is "On MongoDB running in the cloud, version 5.0.8, with a model `users` that has a `String` field `firstName`, a search for `$foo` does not return an entry with exactly that string as the value of `firstName`, but a search for `foo` does." Correct?

**Reply by janpio** [reply]
I think I could reproduce this and will open a bug issue with a reproduction.

**Reply by janpio** [reply]
https://github.com/prisma/prisma/issues/13089

**Reply by TrabelsiHoussem** [reply]
the problem is to select a field which contains a  string with a dollar symbol  .

**Reply by TrabelsiHoussem** [reply]
also a string  starts with 2 dollars generate an error

**Reply by janpio** [reply]
> the problem is to select a field which contains a string with a dollar symbol .

Yes, I reproduced that problem and opened an issue about it: https://github.com/prisma/prisma/issues/13089

**Reply by TrabelsiHoussem** [reply]
thank you

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
